generated/niscope/niscope/session.py

```diff
@@ -4226,8 +4311,8 @@ def configure_horizontal_timing(self, min_sample_rate, min_num_pts, ref_position
         return
 
     @ivi_synchronized
-    def _configure_ref_levels(self, low=10.0, mid=50.0, high=90.0):
-        r'''_configure_ref_levels
+    def configure_ref_levels(self, low=10.0, mid=50.0, high=90.0):
```

Comment:
I had initially made it public since it was on the list of functions that should be exposed in the linked issue, but I've dug around and it seems like it's obsolete because you can instead directly set the attributes managed by configure_ref_levels. I'll move it back to being private.